solidity/contracts/BurntPixArchives.sol

```diff
@@ -99,13 +99,19 @@ contract BurntPixArchives is LSP8CappedSupply {
     }
 
     function refineToArchive(uint256 iters, address contributor) public {
-        require(iters > 0);
+        //should we stop refining if the original is claimed?
```

Comment:
collection might not be minted out :), so there could still exist desire to keep minting just for the sake of having a piece of the idea